### PR TITLE
Turbo/curate

### PR DESCRIPTION
**Pull request by austinFlipside**
- `ez_nft_sales` (quixotic sales)
- `fact_hourly_token_prices` (OP token prices)
- velodrome schema
- `ez_claimed_rewards`
- `ez_lp_actions`
- `ez_pool_details`
- `ez_staking_actions`
- `ez_swaps`
- `ez_velo_locks`
- `ez_votes`
- tests and docs

### CODE REVIEW

File: models/velodrome/velodrome__ez_claimed_rewards.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      - name: ORIGIN_FROM_ADDRESS
+        description: '{{ doc("opt_origin_from") }}'
+      - name: ORIGIN_TO_ADDRESS
+        description: '{{ doc("opt_origin_to") }}'
+      - name: CONTRACT_ADDRESS
+        description: 'The contract address interacted with in this event.'
+      - name: EVENT_INDEX
+        description: 'The event index within the transaction.'
+      - name: REWARD_TYPE
+        description: 'The type of rewards claimed, either `voter_rewards`, `venft_distribution`, or `lp_reward`.'
```
Comment (jhuhnke) [inline models/velodrome/velodrome__ez_claimed_rewards.yml:24]: My misunderstanding on Velo - does a user choose their reward type? Or is this auto chosen for them? 

If user selected, maybe say something like "The type of reward the user elected to claim, ..."

File: models/velodrome/velodrome__ez_claimed_rewards.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      - name: TX_HASH
+        description: '{{ doc("opt_tx_hash") }}'
+      - name: ORIGIN_FUNCTION_SIGNATURE
+        description: '{{ doc("opt_origin_sig") }}'
+      - name: ORIGIN_FROM_ADDRESS
+        description: '{{ doc("opt_origin_from") }}'
+      - name: ORIGIN_TO_ADDRESS
+        description: '{{ doc("opt_origin_to") }}'
+      - name: CONTRACT_ADDRESS
+        description: 'The contract address interacted with in this event.'
```
Comment (jhuhnke) [inline models/velodrome/velodrome__ez_claimed_rewards.yml:20]: For this, do we want to put "this field can be joined to labels to get the contract name where available?"

File: models/velodrome/velodrome__ez_lp_actions.yml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      - name: POOL_NAME
+        description: '{{ doc("op_pool_name") }}'
+      - name: POOL_TYPE
+        description: '{{ doc("op_pool_type") }}'
+      - name: SENDER_ADDRESS
+        description: 'The sender of the liquidity action.'
+      - name: LP_ACTION
+        description: 'The liquidity action, either deposit or withdraw.'
+      - name: TOKEN0_SYMBOL
+        description: 'The symbol of token0 in the pool.'
```
Comment (jhuhnke) [inline models/velodrome/velodrome__ez_lp_actions.yml:32]: Could make the .md file for token0 / 1 symbol more general and put here?

File: models/velodrome/velodrome__ez_staking_actions.yml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+      - name: EVENT_INDEX
+        description: 'The index of this event within the transaction.'
+      - name: STAKING_ACTION_TYPE
+        description: 'The type of staking action, either deposit or withdraw.'
+      - name: LP_TOKEN_AMOUNT
+        description: 'The amount of LP tokens deposited or withdrawn.'
+      - name: LP_PROVIDER_ADDRESS
+        description: 'The address of the liquidity provider.'
+      - name: GAUGE_ADDRESS
+        description: 'The address of the gauge.'
```
Comment (jhuhnke) [inline models/velodrome/velodrome__ez_staking_actions.yml:30]: Quick description of a gauge might be helpful

File: models/velodrome/velodrome__ez_staking_actions.yml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+      - name: POOL_ADDRESS
+        description: 'The address of the liquidity pool.'
+      - name: POOL_NAME
+        description: 'The name of the liquidity pool.'
+      - name: POOL_TYPE
+        description: 'The type of liquidity pool, either stable or volatile.'
```
Comment (jhuhnke) [inline models/velodrome/velodrome__ez_staking_actions.yml:36]: Same as above - threshold for stable / volatile pool

File: models/velodrome/velodrome__ez_votes.yml [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+      - name: ORIGIN_FUNCTION_SIGNATURE
+        description: '{{ doc("opt_origin_sig") }}'
+      - name: ORIGIN_FROM_ADDRESS
+        description: '{{ doc("opt_origin_from") }}'
+      - name: ORIGIN_TO_ADDRESS
+        description: '{{ doc("opt_origin_to") }}'
+      - name: GAUGE_ADDRESS
+        description: 'The address of the gauge in this vote or unvote.'
+      - name: EXTERNAL_BRIBE_ADDRESS
+        description: 'The external bribe address for the gauge used in the event.'
```
Comment (jhuhnke) [inline models/velodrome/velodrome__ez_votes.yml:22]: Another misunderstanding on me + velodrome. Is an external & internal bribe exactly what I expect? Maybe a one sentence primer on Velo bribes would be good.

**Review by jhuhnke — CHANGES_REQUESTED** [review]

File: models/velodrome/velodrome__ez_claimed_rewards.yml (hunk 1 above)
Comment (austinFlipside) [reply models/velodrome/velodrome__ez_claimed_rewards.yml:24]: there are three ways to earn rewards on velodrome - this table encompasses all of them. Reward type will depend on the actions taken by the user (staking LP tokens, locking VELO, or voting on pools with veVELO)

**Review by jhuhnke — APPROVED** [review]

**Pull request merged**